docs/topics/native/native-memory-manager.md

```diff
@@ -54,9 +54,49 @@ build script:
 
 ## Memory consumption
```

Comment:
I think it would be good to have at least one sentence here to introduce the subject matter so that there isn't a sub-heading immediately after a heading. Maybe something like: "With the memory manager it's possible to monitor memory consumption so that you can check for memory leaks or detect other memory problems."